test/common/cdp.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
```

Comment:
Please drop the shebang.  This isn't supposed to be executed directly.

Reply:
Of course, thanks.